src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ParameterTest/ParametersTest.cs

```diff
@@ -902,13 +904,12 @@ public static void ClosedConnection_SqlParameterValueTest()
                             Assert.Fail($"Unexpected exception occurred: {e.Message}");
```

Comment:
This is super redundant ... the test will fail if an unexpected exception is thrown, so there's no need for this. *sigh* I can't expect you to fix all the issues with this code but it really needs help